core/src/main/java/smashdudes/graphics/effects/ParticleEmitter.java

```diff
@@ -74,22 +79,10 @@ public void render(SpriteBatch sb)
 
         for(Particle p : activeParticles)
         {
-            drawParticle(sb, p);
+            p.render(sb);
         }
     }
 
-    private boolean canSpawnParticle()
```

Comment:
I wasn't too sure what the purpose was for this method, so I removed it

Reply:
I would put this back in, make it act more like an iterator and removes the spawnTimer logic from updating

Reply:
I find it confusing to update spawning logic in a function which appears to determine the state of the emitter.

What do you mean by "act more like an iterator"?